docs/standalone/audit_guidelines.md

```diff
@@ -0,0 +1,137 @@
+
+* Hydra Head protocol implementation is immune to attacks via chain transactions.
+* Hydra Head protocol implementation is immune to attacks via network.
+* Hydra Head protocol implementation faithfully reflects the head state through the API.
+* Hydra Head protocol implementation is immune to API attacks.
+* Any attack which would be invalid under the assumptions stated above in the document.
+
+# Artifacts
+
+This sections gives a detailed description of the artifacts mentioned above in the document:
+ - Coordinated Hydra Head V1 Specification
+ - Hydra plutus scripts (on-chain code)
+ - Hydra node chain layer code (off-chain code)
+
+## Artifact 1: Coordinated Hydra Head V1 Specification
+
+The Hydra Head protocol implementation derives from [Hydra: Fast Isomorphic State Channels](https://eprint.iacr.org/2020/299.pdf) in several ways. Especially some simplifications have been introduced and generalizations removed.
+
+The [Coordinated Hydra Head V1 specification](https://docs.google.com/document/d/1XQ0C7Ko3Ifo5a4TOcW1fDT8gMYryB54PCEgOiFaAwGE/) captures these deviations and also includes the "formal notation" of the actual transaction constraints (which are foregone in the original paper). Also, it details the L2 protocol logic for the **Coordinated** Head protocol - which is implemented in V1.
+
+## Artifact 2: Hydra Head Protocol Implementation
+
+With Hydra Head Protocol Implementation we refer to the software component that is used to operate a node in the Hydra Head protocol. The `hydra-node` allows its users to open a head, lock funds in it, connect to peers, process transactions as a layer 2, close a head and unlock the corresponding funds. It is comprised by the Hydra plutus scripts, Hydra head chain layer, layer 2 code, network communication between peers, and an API for clients to connect and use the node.
+
+* Source code repository: [input-output-hk/hydra](https://github.com/input-output-hk/hydra)
+* Version ready to be audited: [0.9.0](https://github.com/input-output-hk/hydra/releases/tag/0.9.0)
+
+The following parts are described below:
+ - Hydra plutus scripts (on-chain code)
+ - Hydra node chain layer code (off-chain code)
+
+### Artifact 2.1: Hydra plutus scripts (on-chain code)
+
+Hydra plutus scripts source code can be found in the [hydra-plutus module](https://github.com/input-output-hk/hydra/tree/master/hydra-plutus).
+
+Hydra plutus mutation based testing can be found in the [hydra-node module](https://github.com/input-output-hk/hydra/tree/master/hydra-node) in test/Hydra/Chain/Direct/Contract/.
+
+### Artifact 2.2: Hydra node chain layer code (off-chain code)
+
+Hydra node chain layer code can be found in the [hydra-node module](https://github.com/input-output-hk/hydra/tree/master/hydra-node) in src/Hydra/Chain/Direct/
+
+Hydra node chain layer tests can be found in the [hydra-node module](https://github.com/input-output-hk/hydra/tree/master/hydra-node) in test/Hydra/Chain/Direct/
```

Comment:
The links here could refer to the directories directly, e.g. [chain layer code](https://github.com/input-output-hk/hydra/tree/master/hydra-node/src/Hydra/Chain)